### PR TITLE
Use the user's language

### PR DESCRIPTION
**Pull request by marcelklehr**

### CODE REVIEW

**Comment by marcelklehr** [comment]
```
Es gibt vier Faktoren, die das "Ethical AI Rating" beeinflussen:
1. Ist das Software (beides für Schätzung und Schulung) unter einer freien und offenen Lizenz veröffentlicht?
2. Ist das ausgebildete Modell kostenlos für Selbstbetrieb zur Verfügung gestellt?
3. Ist das Schulungsdaten kostenlos und frei zur Verfügung?

Je mehr dieser Faktoren erfüllt sind, desto höher ist die Bewertung.
```

Not entirely idiomatic, but good enough. Perhaps we can update the model to llama 3 at some point.

**Comment by kyteinsky** [comment]
> Sources referenced to generate the above response:

we should do something about this, or maybe let it be until NC 30 after which the fields will be separated and the heading would be part of the UI?

> Not entirely idiomatic, but good enough.

don't know German but the translated text was good.

> Perhaps we can update the model to llama 3 at some point.

Was waiting for the tokenizer fix for llama-cpp. It has been a while since that was fixed, let's switch to that soon. :100:

**Comment by marcelklehr** [comment]
> we should do something about this, or maybe let it be until NC 30 after which the fields will be separated and the heading would be part of the UI?

Yeah, I think it's fine for now.

File: context_chat_backend/chain/one_shot.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from .context import ScopeType, get_context_chunks, get_context_docs
 from .query_proc import get_pruned_query
 
-_LLM_TEMPLATE = '''Answer based only on this context and do not add any imaginative details:
+_LLM_TEMPLATE = '''Answer based only on this context and do not add any imaginative details. Make sure to use the same language as the question in your answer.
 {context}
 
 {question}
```
Comment (kyteinsky) [inline context_chat_backend/chain/one_shot.py:11]: Adding `'''  # noqa: E501` near the end of the string should let the pre-commit workflow pass

Comment (marcelklehr) [reply context_chat_backend/chain/one_shot.py:11]: This chain is not used at all anyway though, right?

Comment (kyteinsky) [reply context_chat_backend/chain/one_shot.py:11]: This is the only chain we have. The linked prompt is a fallback if no prompt is given in the config.

**Review by kyteinsky — APPROVED** [review]

**Pull request merged**